src/Sylius/Component/Shipping/Model/ShipmentInterface.php

```diff
@@ -48,7 +48,7 @@ public function getMethod();
     /**
      * @param ShippingMethodInterface $method
      */
-    public function setMethod(ShippingMethodInterface $method);
+    public function setMethod(ShippingMethodInterface $method = null);
```

Comment:
I believe setting `null` should not be possible. We should also prevent from creating Shipment without shipping method on a database level.